src/main/resources/view/DarkTheme.css

```diff
@@ -357,35 +450,89 @@
 }
 
 #availabilities .label {
-    -fx-text-fill: white;
-    -fx-background-color: #B57EDC;
+    -fx-text-fill: accent-blue;
+    -fx-background-color: accent-light-blue;
     -fx-padding: 1 3 1 3;
     -fx-border-radius: 2;
     -fx-background-radius: 2;
-    -fx-font-size: 11;
+    -fx-font-size: 15;
+    -fx-font-weight: 300;
+    -fx-effect: dropshadow(gaussian, lightblue-dropshadow, 13, 0, 0, 0);
 }
 
 #meetingPane{
     -fx-border-radius: 20 20 20 20;
     -fx-background-radius: 20 20 20 20;
-    -fx-background-color: #3d549d;
+    -fx-background-color: accent-yellow;
     -fx-border-width: 0;
     -fx-background-insets: 0 0 0 0;
     -fx-background-padding: 50 100 0 0;
-    -fx-text-fill: white;
-    -fx-effect: dropshadow(gaussian, rgba(62,84,158,0.8), 13, 0, 0, 0);
+    -fx-effect: dropshadow(gaussian, yellow-dropshadow, 13, 0, 0, 0);
  }
 
-#meetingListView {
-    -fx-background-color: transparent;
-    -fx-padding: 7 0 0 0;
-}
+ #nameCard{
+     -fx-border-radius: 20 20 20 20;
+     -fx-background-radius: 20 20 20 20;
+     -fx-background-color: transparent;
+     -fx-border-width: 0;
+     -fx-background-insets: 0 0 0 0;
+     -fx-background-padding: 50 100 0 0;
 
-#meetingListCell {
+  }
+  #nameCard .label {
+
+      -fx-font-family: "Segoe UI Semibold";
+      -fx-font-size: 23px;
+      -fx-text-fill: accent-yellow;
+
+  }
+
+ #meetingListView {
     -fx-background-color: transparent;
     -fx-padding: 5 0 0 0;
-}
+ }
 
-#favourite_label {
-    -fx-text-fill: #FFDF00;
-}
```

Comment:
Yup it was intentional because now the star icon now shares the same styling as the label in the name card!

Reply:
> Yup it was intentional because now the star icon now shares the same styling as the label in the name card!

@Pluiexo I think the purpose of favourite icon is to stand out from the name and index, perhaps you could modify to change the color of the icon? If you still think that it is okay for it to not stand out, then no change is required.

Reply:
Will consider this suggestion well!